packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -175,12 +180,16 @@ export class CommonWell {
    * @param id       The org to be found
    * @returns
    */
-  async getOneOrg(meta: RequestMetadata, id: string): Promise<Organization> {
+  async getOneOrg(meta: RequestMetadata, id: string): Promise<Organization | undefined> {
     const headers = await this.buildQueryHeaders(meta);
     const resp = await this.api.get(`${CommonWell.MEMBER_ENDPOINT}/${this.oid}/org/${id}/`, {
       headers,
+      validateStatus: null, // don't throw on status code > 299
     });
-    return organizationSchema.parse(resp.data);
+    const status = resp.status;
+    if (status === 404) return undefined;
+    if (status >= 200 && status < 300) return organizationSchema.parse(resp.data);
```

Comment:
I'll replace those literals by `http-status`

Reply:
Actually, I'd like to suggest we use literal numbers instead of `http-status`, bc:
- those number won't change
- using numbers is nicer than using codes when we want to do stuff like here (`>=`, `<`)
- there's always some risk when we add a library

Reply:
`status.BAD_REQUEST` is easier to read quickly than `400` -> hard to quantify but it should net save us time with PR review and looking back at legacy code as we won't have to look up HTTP codes

Reply:
> status.BAD_REQUEST is easier to read quickly than 400

I think this is a personal preference. I prefer numeric, I find it easier to remember the meaning of the number than the exact words used to express it in natural language.

How to you suggest we solve the issue of ranges? The solution `http-status` gives is super verbose compared to numeric range verification.

Reply:
Just reintroduced `http-status`. Not too bad and we're already using across apps.